### PR TITLE
Assignment04-LOB33

### PR DESCRIPTION
**Pull request by LuisOB33**
I couldnt find my studentID as number

### CODE REVIEW

File: Assignment4/Luis0B33-luis.ortiz.benito/task06.py [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+
+Automatically generated by Colaboratory.
+
+Original file is located at
+    https://colab.research.google.com/drive/10JP8EmMvffxSWT1YON775OHs4-MLxE45
+
+**Task 06: Modifying RDF(s)**
+"""
+
+!pip install rdflib
```
Comment (dgarijo) [inline Assignment4/Luis0B33-luis.ortiz.benito/task06.py:12]: won't work in Python. This is notebook magic

File: Assignment4/Luis0B33-luis.ortiz.benito/task06.py [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+
+# Visualize the results                #use VCARD to see only new changes
+for s, p, o in g.triples((ns.JaneSmith, None, None)):
+  print(s,p,o)
+
+"""**TASK 6.5: Add UPM as the university where John Smith works**"""
+
+# DONE
+g.add((ns.UPM, RDF.type, ns.University))
+g.add((ns.JonhSmith, VCARD.Work, ns.UPM))
```
Comment (dgarijo) [inline Assignment4/Luis0B33-luis.ortiz.benito/task06.py:73]: Work does not exist in vcard. Create a new property in your ns

File: Assignment4/Luis0B33-luis.ortiz.benito/task07.py [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+
+"""
+
+#DONE
+query2 = prepareQuery("""
+  SELECT ?person WHERE{
+    {
+      ?person a ns:Person.
+    } UNION {
+      ?subclass RDFS:subClassOf ns:Person.
```
Comment (dgarijo) [inline Assignment4/Luis0B33-luis.ortiz.benito/task07.py:54]: need to include all taxonomy (use chained queries with *)

File: Assignment4/Luis0B33-luis.ortiz.benito/task07.py [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+  }
+  """, initNs = {
+      "ns":ns,
+      "RDFS":RDFS
+  }
+)
+
+# Visualize the results
+for r in g.query(query3):
+  print(r.subclass, r.person, r.property)
```
Comment (dgarijo) [inline Assignment4/Luis0B33-luis.ortiz.benito/task07.py:91]: same as above.

Also, missing exercises in rdflib

File: Assignment4/Luis0B33-luis.ortiz.benito/task08.py [hunk 5]
```diff
@@ -0,0 +1,95 @@
...
+
+for r in g1.query(query1):
+  instance = {
+    "person": r.person,
+    "given": r.given, 
+    "family": r.family,
+    "mail": r.mail
+  }
+  print(instance)
+  print(None in r)
```
Comment (dgarijo) [inline Assignment4/Luis0B33-luis.ortiz.benito/task08.py:95]: a little complex, but looks ok

**Review by dgarijo — CHANGES_REQUESTED** [review]

File: Assignment4/Luis0B33-luis.ortiz.benito/task07.py (hunk 4 above)
Comment (LuisOB33) [reply Assignment4/Luis0B33-luis.ortiz.benito/task07.py]: Changes made!

**Comment by dgarijo** [comment]
@LuisOB33,
there are many changes still pending

**Review by dgarijo — APPROVED** [review]

**Pull request merged**